.gitignore

```diff
@@ -6,3 +6,6 @@ publish
 *.wasm
 *.wat
 test.config
+
+# Generated by macOS
+.DS_Store
```

Comment:
FWIW I think this is best stored in your global `~/.gitignore` file rather than having all projects you work on add this entry